src/DynamoCore/PackageManager/PackageManagerClient.cs

```diff
@@ -64,13 +66,15 @@ public bool HasAuthProvider
 
         #endregion
 
-        internal PackageManagerClient(IGregClient client, string rootPackageDirectory, CustomNodeManager customNodeManager)
+        internal PackageManagerClient(IGregClient client, PackageUploadBuilder builder)
         {
-            this.rootPackageDirectory = rootPackageDirectory;
```

Comment:
Check out the new code in DynamoModel.  This is a much cleaner piece of dependency injection.